### PR TITLE
Fix/meta title and insert data in changelog of liquibase

### PR DESCRIPTION
**Pull request by VoNhu2901**
image

### CODE REVIEW

**Comment by LEHOANGGLAM** [comment]
Approved

File: product/src/main/resources/db/changelog/data/changelog-brand.sql [hunk 1]
```diff
@@ -4,4 +4,4 @@
 insert into brand (name, slug) values ('Apple', 'Apple');
 
 --changeset khanhtran:issue-335-3
-insert into brand (name, slug) values ('Samsung', 'Samsung');
\ No newline at end of file
+insert into brand (name, slug) values ('Samsung', 'Samsung');
```
Comment (khoahd7621) [inline product/src/main/resources/db/changelog/data/changelog-brand.sql]: The data for the field slug should be all lowercase.

**Review by LEHOANGGLAM — APPROVED** [review]
approved

File: product/src/main/resources/db/changelog/data/changelog-product.sql [hunk 2]
```diff
@@ -0,0 +1,9 @@
+----liquibase formatted sql
```
Comment (khoahd7621) [inline product/src/main/resources/db/changelog/data/changelog-product.sql:1]: Let's remove this file if all queries are commented out.

**Review by LEHOANGGLAM — CHANGES_REQUESTED** [review]
this PR is duplicated your previous PR. You need to close previous PR first.

File: backoffice/modules/catalog/models/Product.ts [hunk 3]
```diff
@@ -10,6 +10,7 @@ export type Product = {
   gtin: string;
   slug: string;
   price: number;
+  metaTitle?: string;
```
Comment (khoahd7621) [inline backoffice/modules/catalog/models/Product.ts:13]: I am trying to understand why this field is optional.

Comment (VoNhu2901) [reply backoffice/modules/catalog/models/Product.ts]: i resolved all your comment

**Review by khoahd7621 — APPROVED** [review]
Nice, thanks.

**Review by LEHOANGGLAM — APPROVED** [review]

**Pull request merged**